stan/math/rev/functor/cvodes_integrator.hpp

```diff
@@ -81,7 +81,13 @@ class cvodes_integrator {
     using initial_var = stan::is_var<T_initial>;
     using param_var = stan::is_var<T_param>;
 
-    const char* fun = "integrate_ode_cvodes";
+    const char* fun;
+
+    if (Lmm == CV_BDF) {
+      fun = "integrate_ode_bdf";
+    } else {
+      fun = "integrate_ode_adams";
+    }
```

Comment:
This should be a one-liner:

```
const char* fun = Lmm == CV_BDF ? "integrate_ode_bdf" : "integrate_ode_adams";
```

Reply:
This should be an argument to the function haha

Reply:
I made this an argument to the function.